Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1105,7 +1105,11 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
             "enable-webaudio",
             _("Enable WebAudio"),
             _("Whether WebAudio content should be handled"),
+#if ENABLE(WEB_AUDIO)
             FEATURE_DEFAULT(WebAudioEnabled),
+#else
+            TRUE,
```

Comment:
shouldn't this be FALSE?

Reply:
See also the default value for `enable-media`.

But maybe it would make more sense to default to `FALSE` there too if the feature is not enabeld.

Reply:
cc @aperezdc

Reply:
I have changed to `FALSE` for the WebAudio case

Reply:
I agree this should be `FALSE` because when WebAudio is disabled at build time it's not really possible to ever enable it 😉 

I would even argue that we should prevent changing the setting to `TRUE`, and emit a warning.

Reply:
Thanks all for the feedback.
Ok to merge as-is for a first step?

Reply:
> Thanks all for the feedback. Ok to merge as-is for a first step?

go head!